.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: cpp
+
+dist: trusty
```

Comment:
Trusty is the latest Travis CI offers. Beleive me, I tried to get a more recent distro to get better compilers